tv_apps/smart-system/js/browser_context_menu.js

```diff
@@ -204,14 +206,37 @@
     // context menu api
     if (detail.contextmenu && detail.contextmenu.items.length) {
       var that = this;
-      detail.contextmenu.items.forEach(function(choice, index) {
+      detail.contextmenu.items.forEach(function(choice) {
+        var label;
+        var clickAction;
+
+        if (AppInstallManager.isMarketplaceAppActive()) {
```

Comment:
I was still thinking some of sections are related to app_install_manager but some of them are related to browser_context_menu ex: label & clickAction which should belong to browser_context_menu.
IMO, I would like to leave it here